src/App.js

```diff
@@ -8,21 +8,23 @@ import Modal from './Modal'
 import sortMovies from './sort.js'
 import SortDropdown from './SortDropdown'
 import { Route, Link } from 'react-router-dom'
+import SearchBar from './SearchBar'
 
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
       movies: [],
-      currentMovie: {}
+      currentMovie: {},
+      filtered: []
```

Comment:
This allowed us to populate the filtered list with all of the movies again without having to make another get request